src/patternfly/components/MenuToggle/menu-toggle.hbs

```diff
@@ -15,7 +15,9 @@
     pf-m-expanded
   {{/if}}
   {{#if menu-toggle--modifier}} {{menu-toggle--modifier}}{{/if}}"
-  type="button"
+  {{#unless menu-toggle--type}}
+    type="button"
+  {{/unless}}
   {{#if menu-toggle--IsExpanded}}
```

Comment:
Looks like we need to move this into the `{{#unless menu-toggle--type}}` conditional above - `aria-expanded` should only be on the plain button you've added.